contracts/Vault.vy

```diff
@@ -124,6 +179,13 @@ def __init__(
 @pure
 @external
 def apiVersion() -> String[28]:
```

Comment:
Side note: `String[N]` is actually a "dynamic array" in Vyper, however it has an upper limit of `N` so that all dynamic operations have a statically determinable execution cost.

Reply:
I did not know that, thank you!